### PR TITLE
Update apparmor.md

### PR DESCRIPTION
**Pull request by islinwb**
fix typos

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4875)

### CODE REVIEW

File: docs/tutorials/clusters/apparmor.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Where `<container_name>` is the name of the container to apply the profile to, a
 specifies the profile to apply. The `profile_ref` can be one of:
 
 * `runtime/default` to apply the runtime's default profile.
-* `localhost/<profile_name>` to apply the profile loaded on the host with the name `<profile_name>`
+* `localhost/<profile_name>` to apply the profile loaded on the host with the name `<profile_name>`.
```
Comment (zacharysarah) [inline docs/tutorials/clusters/apparmor.md]: Even better would be:
- adding Markdown for an unordered list
- removing the period from this list item and the one above it

File: docs/tutorials/clusters/apparmor.md [hunk 2]
```diff
@@ -370,7 +370,7 @@ explicitly reject the annotations for at least 2 releases after that.
 Getting AppArmor profiles specified correctly can be a tricky business. Fortunately there are some
 tools to help with that:
 
-* `aa-genprof` and `aa-logprof` generate profile rules by monitoring an application's activity and
+* `aa-genprof` and `aa-logprof` generate profile rules by monitoring an application's activities and
```
Comment (zacharysarah) [inline docs/tutorials/clusters/apparmor.md]: "Activity" is good English; does it increase technical accuracy to change it to "activities"?

File: docs/tutorials/clusters/apparmor.md [hunk 3]
```diff
@@ -407,7 +407,7 @@ Specifying the profile a container will run with:
     containers, and unconfined (no profile) for privileged containers.
 - `localhost/<profile_name>`: Refers to a profile loaded on the node (localhost) by name.
   - The possible profile names are detailed in the
-    [core policy reference](http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference#Profile_names_and_attachment_specifications)
+    [core policy reference](http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference#Profile_names_and_attachment_specifications).
```
Comment (zacharysarah) [inline docs/tutorials/clusters/apparmor.md]: 👍

File: docs/tutorials/clusters/apparmor.md [hunk 4]
```diff
@@ -423,7 +423,7 @@ Specifying the list of profiles Pod containers is allowed to specify:
 * **key**: `apparmor.security.beta.kubernetes.io/allowedProfileNames`
 * **value**: a comma-separated list of profile references (described above)
   - Although an escaped comma is a legal character in a profile name, it cannot be explicitly
-    allowed here
+    allowed here.
```
Comment (zacharysarah) [inline docs/tutorials/clusters/apparmor.md]: 👍

**Comment by zacharysarah** [comment]
@islinwb :wave: Thanks for opening this PR. Please check the box to allow edits from maintainers.

Allow edits from maintainers checkbox

File: docs/tutorials/clusters/apparmor.md (hunk 2 above)
Comment (islinwb) [inline docs/tutorials/clusters/apparmor.md]: Many thanks for your helpful suggestions! It doesn't increase technical accuracy as I thought by changing "activity" to  "activities". I just made some modifications accordingly.

**Pull request merged**